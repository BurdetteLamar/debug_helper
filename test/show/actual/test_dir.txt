---
Dir (message='test_dir'):
  Dir#path: C:/Users/Burde/Documents/GitHub/debug_helper/test
  Dir.entries("C:/Users/Burde/Documents/GitHub/debug_helper/test"):
  - "."
  - ".."
  - debug_helper_test.rb
  - show
  - show_hash
  - test_helper.rb
  Dir.exist?("C:/Users/Burde/Documents/GitHub/debug_helper/test"): true
