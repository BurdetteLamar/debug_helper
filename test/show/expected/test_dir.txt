---
Dir (message='test_dir'):
  Dir#path: C:/Users/Burdette/Documents/GitHub/debug_helper/test
  Dir.entries("C:/Users/Burdette/Documents/GitHub/debug_helper/test"):
  - "."
  - ".."
  - debug_helper_test.rb
  - show
  - show_hash
  - test_helper.rb
  Dir.exist?("C:/Users/Burdette/Documents/GitHub/debug_helper/test"): true
